modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-type-fieldset/src/main/resources/META-INF/resources/fieldset.soy

```diff
@@ -11,10 +11,9 @@
  * Prints a column.
  */
 {template .fieldset_column}
-	{@param columnSize: int}
```

Comment:
Hey @pedroqueiroz94 , @rafaprax would you guys see in this file: https://github.com/clovisdasilvaneto/liferay-portal/blob/master/modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-type-fieldset/src/main/java/com/liferay/dynamic/data/mapping/type/fieldset/internal/FieldSetDDMFormFieldTemplateContextContributor.java if the backend still parsing the columnSize paramiter to the fieldset.soy file?

Reply:
Hi @pedroqueiroz94, should I remove this columnSize parameter on on this file since we're not using anymore to create a class name?

Reply:
Hey guys, I'm not sure we should remove this. Although the only place we use FieldSets is in data provider and rules, it might be used in other places in the future. Without this, we should not be able to allow different field sizes inside a field set, is that right?

@leoadb @rafaprax, what's your opinion on this?